src/app/auth/duck/operations.ts

```diff
@@ -0,0 +1,49 @@
+import fetch from "cross-fetch";
+import { Creators } from "./actions";
+import { history } from "../../common/duck/utils";
+
+const requestTokenAction = Creators.requestToken;
+const receiveTokenAction = Creators.receiveToken;
+const failedTokenRequest = Creators.failedToken;
+// token cmd from terminal
+//TOKEN=$(kubectl get secrets \
+//  -o jsonpath='{.items[?(@.type=="kubernetes.io/service-account-token")].data.token}' \
+//  | base64 --decode)
+// APISERVER=$(kubectl config view -o \
+//  jsonpath='{.clusters[*].cluster.server}')
+const loginRequest = (username, password) => {
+  return dispatch => {
+    dispatch(requestTokenAction());
+    //simulate token request
+
+    const mockLoginSuccessful: TimerHandler = () => {
+      const hardCodedToken =
+        "eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJkZWZhdWx0Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6ImJ1aWxkZXItdG9rZW4tNWdxdmgiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoiYnVpbGRlciIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6IjY4Nzc3YzdjLTNmNjctMTFlOS04ZjJlLTBlMmRiNjE2Y2JjOCIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDpkZWZhdWx0OmJ1aWxkZXIifQ.nOSiUJDgsVljF8uc4yMAZotnNE05-P2OmWck8I-loOnoiiZlEnU0leSv91zeoe13wkgzR4d5aGzRz2PxXdGkYQXWZx9wk85XHcH7bE5kPt4dDXIKE6InvQozurfae4syRu5JpVSzPYxG1L777mm6CEvVuADSwXm6PqR5xPtICLPgUZos6flvtB9ne6eIJ8TwYyh1XE3W_eLLWXZoOhNUmF3QO4rtXV2AvMXS6zLgaPXbVeyHqDy9cFSaFqpzzbQ5KRrVGL0z8ME3ivHzUMOzU9PoOaW95aSGAtUIcIDQM0b2_5lB-CUDf-KLFSzkeq4wPwwepZka2KzuhnA92ggbQw";
```

Comment:
Can we get this read in dynamically with webpack env vars or a config file that's not checked in until we get this implemented properly?

Reply:
updates